src/structured_data.rs

```diff
@@ -166,10 +166,8 @@ impl StructuredData {
         let data = try!(self.data_to_sign());
         // Count valid previous_owner_signatures and refuse if quantity is not enough
 
-        let check_all_keys = |&sig| {
-            owner_keys.iter()
-                .any(|pub_key| sign::verify_detached(&sig, &data, pub_key))
-        };
+        let check_all_keys =
+            |&sig| owner_keys.iter().any(|pub_key| verify_detached(&sig, &data, pub_key));
```

Comment:
Can this not fail for NO_OWNER_PUB_KEY and still pass overall because 1/2 the vote was ok - or do vaults do something special ?